paymentsheet/src/main/java/com/stripe/android/lpmfoundations/luxe/TransformSpecToElements.kt

```diff
@@ -114,6 +114,6 @@ internal class TransformSpecToElements(
                 is CashAppPayMandateTextSpec -> it.transform(arguments.merchantName)
                 is KlarnaMandateTextSpec -> it.transform(arguments.merchantName)
             }
-        }.takeUnless { it.isEmpty() } ?: listOf(EmptyFormElement())
```

Comment:
This was originally introduced here: https://github.com/stripe/stripe-android/pull/5423

But seems very much like the wrong fix.